obaker1/hungryhelpers
# feature/populateDB

Testing instructions:
- Run 'pip install openpyxl'
- Run 'pip install django-extensions'

This feature is not testable in the django testing environment since a separate testing database is generated when launched. However, testers may test it live by:
1. Running 'python manage.py runscript populateDB'
2. Logging into the admin account created by the script 
3. View the locations in http://127.0.0.1:8000/findLocation/ to verify that the database was successfully populated.

Description of edits:
- Added script that automatically populates the database with necessary information including:
  - locations
  - origin
  - superuser account.
